relay/kusama/constants/src/lib.rs

```diff
@@ -110,7 +110,14 @@ pub mod system_parachain {
 
 	frame_support::match_types! {
 		pub type SystemParachains: impl Contains<MultiLocation> = {
-			MultiLocation { parents: 0, interior: X1(Parachain(ASSET_HUB_ID | ENCOINTER_ID | BRIDGE_HUB_ID)) }
```

Comment:
@svyatonik actually, this is also no good, because this `type SystemParachains` is used in relay chain and from the relay's point-of-view `parents: 0` is correct :)

that PR adds copy of `pub type SystemParachains` directly to the asset-hub or bridge-hub, maybe we could create here types:
```
pub type SystemParachainsWithParent0 // better name
pub type SystemParachainsWithParent1 // better name
```

Reply:
Thanks @bkontur ! I get it now :)